.webauto-ci/main/autoware-build/run.sh

```diff
@@ -23,6 +23,9 @@ if [ -n "$CCACHE_DIR" ]; then
     ccache -M "$CCACHE_SIZE"
 fi
 
+# install xmlschema<4.0.0 before rosdep install as workaround for scenario_simulator_v2
+sudo pip3 install xmlschema==3.4.5
```

Comment:
Do we have to run it with sudo?

```suggestion
pip install --user xmlschema==3.4.5
```

Reply:
at least, it didn't work without sudo. Let me check with `--user`

Reply:
Testing it in here: https://evaluation.ci.tier4.jp/evaluation/reports/042a7547-4dca-57ea-a18b-f1be14a96c5e?project_id=awf

Reply:
It installed it and the [build has succeeded](https://evaluation.ci.tier4.jp/evaluation/reports/042a7547-4dca-57ea-a18b-f1be14a96c5e/builds/15cbe826-949f-545b-b8b0-d6330e640432?project_id=awf).

```
Collecting xmlschema==3.4.5
  Downloading xmlschema-3.4.5-py3-none-any.whl (418 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 418.2/418.2 KB 13.7 MB/s eta 0:00:00
Collecting elementpath<5.0.0,>=4.4.0
  Downloading elementpath-4.8.0-py3-none-any.whl (243 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 243.3/243.3 KB 34.2 MB/s eta 0:00:00
Installing collected packages: elementpath, xmlschema
  WARNING: The scripts xmlschema-json2xml, xmlschema-validate and xmlschema-xml2json are installed in '/home/autoware/.local/bin' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
  WARNING: The scripts xmlschema-json2xml, xmlschema-validate and xmlschema-xml2json are installed in '/home/autoware/.local/bin' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
```

Tests are failing, downloaded logs of one and it says:

```
[openscenario_interpreter_node-3] [ERROR] [launch]: Caught exception in launch (see debug for traceback): Included launch description missing required argument ...
```

So I'm guessing '/home/autoware/.local/bin'  is not on PATH and launch tool cannot access it?

If that's the case, let's keep it with `sudo`.

Reply:
It was due to another issue where autoware_launch branch was pointing to a wrong branch in autoware.repos.
I rerun the test in https://evaluation.ci.tier4.jp/evaluation/reports/5136f9e5-989a-5ae1-bc6d-156ddcc9e6ea?project_id=awf. The scenarios are passing now.

Reply:
applied in https://github.com/autowarefoundation/autoware/pull/6064/commits/32c3fbd3a6c8e797481230d94fde06aae32a8672